proposals/0029-cooperative-vector.md

```diff
@@ -312,8 +312,9 @@ The two input vectors are specified via **input vector 1** and **input vector
 
 The matrix is accumulated to the writeable raw-buffer specified by **matrix
 resource**, with **matrix offset**, **matrix interpretation**, **matrix
-layout**, and **matrix stride** behaving as described
-[above](#matrix-vector-multiply-and-multiply-add-operations).
+layout**, and **matrix stride** behaving as described 
+[above](#matrix-vector-multiply-and-multiply-add-operations), with the constraint
+that **matrix layout** must be `DXILMatrixLayout::OuterProductOptimal`.
```

Comment:
Rather than trying to pack all of this into one sentence, I suggest adding details about the constraints on matrix layout below.  There's always paragraphs talking about constraints on the address of the resource.

I also notice that it refers to the matrix stride needing to be 16-byte aligned, which isn't compatible with requiring OutputProductOptimal layout.

Reply:
Right good point, the matrix stride field itself is currently present only for future proofing for a higher tier that might support Row/column layouts for OPA. Does an update with "matrix stride should be '0' and will be ignored" for TIER_1_1, as this tier only support training_optimal layout for this function" sound ok?

Reply:
Maybe something like "Since only optimal layouts are currently supported for this operation, matrix stride must be 0"?